common/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Mail_Domain.php

```diff
@@ -12,6 +12,7 @@
 $L['Update domain `${0}`'] = 'Update domain `${0}`'; 
 $L['domain_label'] = 'Domain';
 $L['valid_relay_notprimarydomain'] = 'The local mailbox domain cannot be relayed';
+$L['DomainMustNotMatchFQDN'] = 'The domain "${0}" cannot be named like the fully qualified domain name';
```

Comment:
I'd prefer a "positive" form, and refer to the module name in the left menu of Server Manager. I.e.

     The domain name must differ from the server name (FQDN)

Reply:
ok